### PR TITLE
fix remove liquidity issues

### LINKED ISSUE #1203: link in Pool tab moves when hover over it
**Describe the bug**
In the Pool tab, the link "100% Max" changes its place when hovered over.

**To Reproduce**
1. Go to Asset
2. Click on Pool tab
3. Scroll down to Remove Liquidity (make sure you have added liquidity to the pool)
4. Hover over "100% Max"

**Screenshots**
Screen Shot 2022-03-17 at 7 13 23 AM

### LINKED ISSUE #1236: Pool Tab: The Remove button is active even if the user doesn't have oceans in the pool
After successful liquidity removal (Mumbai testnet) from the pool, the interface is confusing since I still see an active remove button (even though I have no more ocean in the pool from this account) and there is NAN % MAX displayed instead of 0

Screenshot 2022-03-17 at 11 24 10

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
- Fixes #1236 
- Fixes #1203 

Changes proposed in this PR:

- reset slider after transaction
- fix NaN error on max button
- fix issue on remove button hover

### CODE REVIEW

**Comment by EnzoVezzaro** [comment]
I'm still experiencing this issue in the Pool tab. It's making my tests really hard 😅. I remember there's an issue opened about it. Do you know if it has been fixed?

https://user-images.githubusercontent.com/13741335/160136328-aa89c44b-d77d-468b-bb36-074b8944b08c.mp4

File: src/components/@shared/atoms/Button.module.css [hunk 1]
```diff
@@ -26,14 +26,14 @@
...
   margin-right: 0;
+  min-width: auto;
 }
 
 .button:hover,
 .button:focus {
   color: var(--brand-white);
   background: var(--brand-grey-light);
   text-decoration: none;
-  transform: translate3d(0, -0.05rem, 0);
```
Comment (kremalicious) [inline src/components/@shared/atoms/Button.module.css:36]: easy fix but this slight movement when a button is hovered is intended, combined with the subtle change in box-shadow. As we always encounter this with buttons with `style="text"` we should rather on that definition overwrite this here and do `transform: none`

Comment (EnzoVezzaro) [reply src/components/@shared/atoms/Button.module.css:36]: got it 👍🏽

**Comment by mihaisc** [comment]
That issue is probably related to the timer in the pool data provider

**Comment by EnzoVezzaro** [comment]
Found and solved the issue about the share amount going to 0, but I'm still having some issues when fetching the new data (after the transaction). Basically, for some reason the data is not getting updated immediately, instead I get the new updated data after 30sec or 1min. 

https://user-images.githubusercontent.com/13741335/160389025-45d5a468-a71d-4c33-9958-8bb4b3572c79.mp4

**Comment by mihaisc** [comment]
Yep, the fun timer

**Review by mihaisc — APPROVED** [review]

**Pull request merged**